### PR TITLE
Include executable hash when submitting multiplayer scores

### PR DESCRIPTION
**Pull request by peppy**
Temporary measure to avoid scores being submitted from an earlier build where there was a breaking score change made.

Blocking until osu-web changes are made and can be tested.

### CODE REVIEW

**Comment by smoogipoo** [comment]
Is this going to handle correctly for mobile devices?

**Comment by peppy** [comment]
No idea!

**Comment by smoogipoo** [comment]
Let me rephrase: are we currently storing the hashes of all builds (macOS/Windows/Linux/iOS/Android) for the osu-web check to see them?

**Comment by peppy** [comment]
We can store any hashes required, yes.

**Comment by peppy** [comment]
I'll test ios / android now to make sure the hash is being retrieved correctly.

**Comment by peppy** [comment]
Seems to work as expected, but I did notice that we should probably be hashing `osu.Game.dll` as executables for net core are regularly just wrappers and may not change between builds.

In the future we will also need to consider hashing individual rulesets as they may have their own changes, but for now `osu.Game.dll` should suffice, based on how we do releases.

**Review by smoogipoo — APPROVED** [review]
